src/Homie/Boot/BootNormal.cpp

```diff
@@ -472,7 +472,7 @@ void BootNormal::_advertise() {
     {
       char* safeConfigFile = Interface::get().getConfig().getSafeConfigFile();
       packetId = Interface::get().getMqttClient().publish(_prefixMqttTopic(PSTR("/$implementation/config")), 1, true, safeConfigFile);
-      free(safeConfigFile);
+      delete safeConfigFile;
```

Comment:
Can anyone else comment on this? My C++ skills are not good enough to review this specific change, sorry.

Reply:
Ok, understood. The safeConfigFile is created in https://github.com/homieiot/homie-esp8266/blob/60ca263c1c39051bc62dd4e96942067062f4fd3e/src/Homie/Config.cpp#L167

and 'delete' is the correct method to release memory when objects are created by the "new" expression (https://www.geeksforgeeks.org/delete-in-c/).

LGTM!